tests/unit/unit_tests.h

```diff
@@ -8,5 +8,6 @@
 /* Name of SplinterDB device created for unit-tests */
 #define TEST_DB_NAME "unit_tests_db"
 
-#define Mega (1024UL * 1024UL)
+#define Kilo (1024UL)
```

Comment:
There are already KiB, MiB, and GiB #defines from platform.  Can we not just use those?